Answers.md

```diff
@@ -1,2 +1,28 @@
 # Your responses to the short answer questions should be laid out here using Mark Down.
 ### For help with markdown syntax [Go here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+
+1. .forEach merely iterates through an array while .map actually iterates and takes the elements to make a new array.
+
+2. Boolean, Null, Undefined, Number, String, Symbol and Object. An array is an object but it comes with tons of methods (like .length) that can be used to iterate through it.
+
+3. Closures are functions called within themselves that can store the updated variables of their outer scoped function. 
```

Comment:
Your definition of closures is 100% correct! I forgot to give you a link to understand it more in-depth but better late than never right?? 😅 
http://javascriptissexy.com/understand-javascript-closures-with-ease/